packages/backend/discovery/_templates/succinct/SP1Blobstream/template.jsonc

```diff
@@ -2,7 +2,49 @@
   "$schema": "../../../../../discovery/schemas/contract.v2.schema.json",
   "ignoreMethods": ["state_dataCommitments", "proxiableUUID"],
   "ignoreInWatchMode": ["latestBlock", "state_proofNonce"],
+  "description": "The Blobstream DA bridge. This contract is used to bridge data commitments between Celestia and Ethereum.",
   "fields": {
+    "$admin": {
+      "handler": {
+        "type": "accessControl",
+        "pickRoleMembers": "TIMELOCK_ROLE"
+      }
+    },
+    "guardians": {
+      "target": {
+        "permissions": [
+          {
+            "type": "configure",
+            "description": "can freeze the bridge contract and update the list of authorized relayers."
+          }
+        ]
+      },
+      "handler": {
+        "type": "accessControl",
+        "pickRoleMembers": "GUARDIAN_ROLE"
+      }
+    },
+    "relayers": {
+      "target": {
+        "permissions": [
+          {
+            "type": "configure",
+            "description": "can call commitHeaderRange() to commit block ranges."
+          }
+        ]
+      },
+      "handler": {
+        "type": "hardcoded",
+        "value": ["0x44eb418a966ff47f5af6f48aea6afde0bf193a8d"]
```

Comment:
Why is this hardcoded?

Reply:
I've migrated handlers as-is from previous config copies - cc @vincfurc